aliyun/src/main/java/org/apache/iceberg/aliyun/oss/BaseOSSFile.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aliyun.oss;
+
+import com.aliyun.oss.OSS;
+
+abstract class BaseOSSFile {
+  private final OSS client;
+  private final OSSURI uri;
+
+  BaseOSSFile(OSS client, OSSURI uri) {
+    this.client = client;
+    this.uri = uri;
+  }
+
+  public String location() {
```

Comment:
I don't see this used. Can you remove it so you don't need to modify `OSSURI` in this commit?

Reply:
it is the implementation of method location for the interface inputfile.

Reply:
Should there be an `@Override` annotation then?

Reply:
Okay, sounds good then.